app/src/main/java/org/novinger/jason/demoreader/ArticleListActivity.java

```diff
@@ -73,30 +59,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
-    private ArrayList<Article> getArticles() {
-        String url = getFeedUrl();
-        OkHttpClient client = new OkHttpClient();
-        ArrayList<Article> articles = new ArrayList<>();
-
-        try {
-            Request request = new Request.Builder().url(url).build();
-            Response response = client.newCall(request).execute();
-            String json = response.body().string();
-            Gson gson = new Gson();
-            Article[] articles_array = gson.fromJson(json, Article[].class);
-            Collections.addAll(articles, articles_array);
-        } catch (IOException ex) {
-            // nothing to see here
-        }
-
-        return articles;
-    }
 
     private String getFeedUrl() {
-        if(qty > 0) {
-           return feedUrl + "?quantity=" + Integer.toString(qty);
+        if(mQuantity > 0) {
+           return mFeedUrl + "?quantity=" + Integer.toString(mQuantity);
```

Comment:
so this is totally fine, and probably how i'd do it, but fwiw there's a lot of old fuddies around that love `String.format`, e.g., `String.format("some string %s", someVariable)`, which has a lot of history, docs, and tricks since it's such a common pattern...